unitary/alpha/quantum_effect.py

```diff
@@ -124,9 +124,13 @@ def then(self, effect: "QuantumEffect"):
         """Use `apply(effect)` instead."""
         return self.apply(effect)
 
-    def apply(self, effect: "QuantumEffect"):
-        """Applies a QuantumEffect conditionally to the specified qubits."""
-        self.then_effect = effect
+    def apply(self, *effects: "QuantumEffect"):
+        """Applies a QuantumEffect conditionally to the specified qubits.
+
+        If multiple effects are specified, they will all be applied
+        (to the same qubits).
```

Comment:
Should we have a TODO() or a suggestion here for cases where effects are applied to different qubits?